sql/moz-fx-data-shared-prod/telemetry_derived/experiment_search_aggregates_v1/query.sql

```diff
@@ -3,95 +3,137 @@ WITH desktop AS (
     submission_timestamp,
     unnested_experiments.key AS experiment,
     unnested_experiments.value AS branch,
-    unnested_ad_clicks.value AS ad_clicks_count,
-    unnested_search_with_ads.value AS search_with_ads_count,
-    unnested_search_counts.count AS search_count,
+    SUM(
+      (
+        SELECT
+          SUM(value.value)
+        FROM
+          UNNEST(payload.processes.parent.keyed_scalars.browser_search_ad_clicks) AS value
+      )
+    ) AS ad_clicks_count,
+    SUM(
+      (
+        SELECT
+          SUM(value.value)
+        FROM
+          UNNEST(payload.processes.parent.keyed_scalars.browser_search_with_ads) AS value
+      )
+    ) AS search_with_ads_count,
+    SUM(
+      (
+        SELECT
+          SUM(`moz-fx-data-shared-prod`.udf.extract_histogram_sum(value.value))
+        FROM
+          UNNEST(payload.keyed_histograms.search_counts) AS value
+      )
+    ) AS search_count
```

Comment:
All historical data in this derived table is basically incorrect. So this should be backfilled. I think backfilling a couple of months back should be sufficient, I'm not aware of anything, except of Grafana depending on this data.